### ISSUE TITLE
Recaptcha v3 always returns a 0.1 score

### ISSUE THREAD

**Issue by SeriaWei** [google/recaptcha#248]
When i use recaptcha v3 with host "www.recaptcha.net" the score is always 0.1, but when i change to use host "www.google.com", the score is 0.9.
So is it the v3 can not use with www.recaptcha.net?
Scripts
```
https://www.recaptcha.net/recaptcha/api.js?render={key}
```
Siteverify
```
https://www.recaptcha.net/recaptcha/api/siteverify
```

**Comment by jasonfedor** [comment]
Thanks for the report - not able to reproduce this, do you have a sample page you ran this? What happened when you mixed and matched siteverify and api.js locations?

**Comment by SeriaWei** [comment]
No, it was just localhost. I will push it to live to test later. Thanks

**Comment by SeriaWei** [comment]
Push to live is the same, the score is always 0.1.
This is my test keys:
### Site key
```
6LcfPWgUAAAAAE89bhoWFMH9Z2xIqinH6MzWxKdz
```
### Secret key
```
6LcfPWgUAAAAABLA0IRBd40FnaqGn4M73IjcQn8t
```

#### With "www.recaptcha.net"
![image](https://user-images.githubusercontent.com/6006218/44069196-3fe84e18-9fb0-11e8-9636-553cccfdf918.png)

#### Change to "www.google.com"
![image](https://user-images.githubusercontent.com/6006218/44069246-6f7b2ab0-9fb0-11e8-86c3-3f4d924c8ad5.png)

But when i change to use **another key**, it works perfect with both "www.recaptcha.net" and "www.google.com"

**Comment by SeriaWei** [comment]
@rowan-m Thanks! But it was just a testing key. :)

**Comment by SeriaWei** [comment]
After change the key, it works at start. But i found that the score would continue to drop from 0.9 until 0.1, and then it would have been 0.1.

**Comment by amarnathm** [comment]
I'm also getting 0.1. But it's on localhost.

**Comment by bankeris** [comment]
Hello, I'm getting always score 0.1 on developer PC only. No matter it's localhost or normal address, it's always 0.1. If i try from other pc/mobile/tablet works like a charm

**Comment by rowan-m** [comment]
If you are running automated tests in a staging/development environment, I would suggest creating a mock service for reCAPTCHA. I updated the various [`RequestMethod` classes](https://github.com/google/recaptcha/blob/master/src/ReCaptcha/RequestMethod/Post.php#L49) to accept a `$siteVerifyUrl` parameter meaning you can override the default location. Alternatively you could also (create a `RequestMethod` class)[https://github.com/google/recaptcha/blob/master/ARCHITECTURE.md#alternate-request-methods] that just returns a hardcoded response.

**Comment by ykcai** [comment]
I am seeing this when I try on Safari, but Chrome returns 0.9. Also testing on localhost. Could localhost be the problem here?

**Comment by ismakv** [comment]
Same thing.
On localhost it was 0.9 after day it become 0.1
On remote host same issue

**Comment by marcotibben** [comment]
I've never used localhost; so for me that's not the problem.. it feels more like google is putting some data together overnight and does some calculations, and after those decides that i'm an abuser filling in those forms with a same kind of message (for my costumers, saying ReCaptcha is activated on their website)..

I've deleted the key and created a new key, and now i'm a 0.9 again..

**Comment by rowan-m** [comment]
Scores you see returned on development machines are not reflective of production. If you need a predictable score returned in a given environment, then you should absolutely mock the service to return a configurable score. To determine where to set appropriate thresholds to act on the score, you need to look at your [dashboard in the admin interface for your site](https://www.google.com/recaptcha/admin) and see the spread of scores being returned.

The score returned is also only an indicator of confidence in that request. If it's below your threshold then you need to determine the appropriate response. For example, if it's just viewing a product page maybe you want to rate limit the user but if it's something like completing a purchase maybe you want to add an SMS verification before letting the user continue.

**Comment by marcotibben** [comment]
Well its happening again. 2 weeks in production and suddenly ReCaptcha keeps returning 0.1 for most of the requests even genuine ones. Customers are complaining. Apparently the machine learning is not good enough yet. I cannot keep creating new keys, just to keep the ReCaptcha performance up to scratch. V3 doesn't seem ready yet. Also I use actions, but most of the sites do not pick the actions up in the admin panel. Some do, but then just one action of the 4 I use.. This is not usable sadly.. It looked very promising, but its not delivering at the moment.

**Comment by jasonfedor** [comment]
Marco, please send your public site key and a url on which you're experiencing trouble to support@recaptcha.net so we can take a look.

**Comment by crmpicco** [comment]
I have implemented reCAPTCHA v3 in our application and we are looking to move this into production soon. I am, however, concerned about some of the scores it returns and a handful of the comments I have read online about it. The [comment above](https://github.com/google/recaptcha/issues/248#issuecomment-443542292) from @marcotibben in particular:

One of the points I am concerned with is the ability to test failures. There doesn’t appear to be a lot of advice in the official docs on how to test this accurately, so what I done was change my User Agent to “Googlebot” and then call the reCAPTCHA API. Initially this returned a failure with a score of 0.1, which is below our pre-defined threshold of 0.5 and would denote bot behaviour. However, now when I do this it returns a score of 0.9 and sometimes 0.999999999. This leads me to believe that Google is in some way whitelisting our staging server IP and returning an acceptable score regardless of what we do.

It would obviously be bad for bots to get through, but I’m more concerned about valid users having a score of 0.1 which was reported on that Github issue a day ago.

Can you advise on recommended testing procedures and how we can go about getting some confidence in this before we think about deploying this to production?

**Comment by marcotibben** [comment]
it was bad for a couple of days, but it seems to have restored now and results are good again. Maybe it's part of the machine learning process?!?

I've made changes to my application in that I don't solely rely on recaptcha's opinion anymore.. bad score resulted in not sending e-mail from contact forms. Now I take the opinion of ReCaptcha in advice and thén decide on some other factors in the message if it's spam or not.. and I log all contact, replies, and failures from recaptcha in my database, so learn where it works and doesn't to improve my application.. I was hoping google would do all the work ;)

I sure hope ReCaptcha gets more reliable, so I can fully rely on it again, because the idea is great..

**Comment by mostafaznv** [comment]
I have this problem too.
recaptcha returns always 0.1 on chrome. but everything is ok in firefox.
I tested it a lot in chrome and I think it returns 0.1 for this reason.

But the main question is this:
if recaptcha analysis my actions and blah blah, why he thinks I'm a robot?

**Comment by doekia** [comment]
Seems the same problem still occur.
How can you could deliver to public release with such major issue ?

**Comment by justid** [comment]
I can confirm this issue which the score keeps on 0.1 even user didn't visit our page before.
After regen the sitekey it return score 0.9, that's ok,but i don't think keep changing the sitekey is a good idea.
Any advice will be appreciated

**Comment by crmpicco** [comment]
I'd be quite keen to read some reviews or blog posts on someone who has successfully integrated reCAPTCHA v3 and have released it to production with a positive outcome. It does appear that people are reluctant to deploy this into a production environment at the moment, certainly, I am one of those people.

**Comment by justid** [comment]
I will using reCAPTCHA v3 into our production environment, and after reading the doc repeatedly,I found these text maybe important:

> reCAPTCHA works best when it has the most context about interactions with your site, which comes from seeing both legitimate and abusive behavior

And I will follow this lead in code, I will update the progress as soon as possible, our production has very large network flow and i think it's good for recaptcha's analytic model

**Comment by neha425** [comment]
can anyone help me for adding recpatcha v3

**Comment by andrebruton** [comment]
I feel like going back to V2 if the V3 issue is not resolved for localhost. It's frustrating to test and fix if there is suddenly a low score. At least with V2 you can make sure you have a high score.

**Comment by maxex0000** [comment]
I have the same issue in my site.
After a dramatic climbing of suspicious rate someday, most results become 0.1.

Is there a mechanism to train the model by site or by key?
If yes, maybe the unsupervised learning mechanism needs to be improved.

**Comment by chxj1992** [comment]
+1 , I load recaptcha v3 from a hidden plain web page in my iOS App, and the score drop from 0.9 until 0.1. Any idea to solve my problem ?   🤔

**Comment by gingerlime** [comment]
bumped into this today as well. We switched to `www.recaptcha.net` recently. One of our employees was getting 0.1 scores, even though she was logged-in on Chrome, no incognito or any crazy browser extensions etc... Weird.

**Comment by CodemasterJamal** [comment]
I'm having the same issue. I'm trying to test a code on my server and google keeps returning a low score, saying that I'm a bot. I really like v3 but, if I have to, I'll have to use v2.

**Comment by mennokreukniet** [comment]
I'm having this issue as well. May be a little different I'm not sure. 

When I log in using autofill on my form, google always gives me a 0.1 score.
If I wait about 10 seconds after autofill I always get a 0.7 score.
If I don't wait, but change something in the autofilled field, I get a 0.7 score as well.

I assume this isn't supposed to happen though? Many people would use autofill to login..

**Comment by ltctech** [comment]
One page on my site nearly always returns 0.1 for me. Another page returns 0.7 or 0.9. I am at a loss.

**Comment by FriendOfTheNight** [comment]
Same issue here. A few months into production and my client's customers are complaining about the failed captcha check. Legitimate human actions are getting a score of 0.1 and blocking users from doing anything (I was blocking anything under 0.3). I had to reduce it to only block a score of 0 for now which I fear may let some bots through. It seems this should automatically give the user a second chance with an "I'm not a Robot" checkbox if there are going to be so many false positives...

**Comment by chenxinlong** [comment]
Similar problem here. When I specify the action parameter as `homepage`, I always got a 0.9 score with chrome, but under safari, it's  a 0.1.

**Comment by CodemasterJamal** [comment]
Hey guys, I kinda solved this issue on stackoverflow (a question in which I asked my self and then answered it on my own). It took a while but, depending on what type of method you are using the recaptcha works well and is relatively easy to use. If you need any assistance feel free to either message me here or contact me on Stackoverflow or Instagram. I'm known as Codemaster Jamal. You can honestly just google that to find me.

My problem was that I had two tags with the same id. Once, I changed the id's and made sure to rewrite the javascript to post to the second-tag, I was fine.

**Comment by marcotibben** [comment]
you didnt kinda solve it.. not using any same id tags here...

**Comment by CodemasterJamal** [comment]
> you didnt kinda solve it.. not using any same id tags here...

You will need to supply some coding examples if you need help. Also, I'll overlook this entire topic and then respond to them the best I can. You also need to specific what type of languages you are using and how you are implementing this method. I use a PHP and Javascript method for my website.

**Comment by marcotibben** [comment]
the recaptcha code the end user uses in his browser is always javascript.. so that doesn't really matter now does it.. and if you had read entire topic you would also have read it works for two weeks, and then suddenly it doesn't, and then it does again.. and then doesn't.. without changing any code.. so the problem more seems to be an unreliable recaptcha solution, than any form of misuse of the code.. if it was as easy as using double id's there wouldn't be a problem..

**Comment by CodemasterJamal** [comment]
> the recaptcha code the end user uses in his browser is always javascript.. so that doesn't really matter now does it.. and if you had read entire topic you would also have read it works for two weeks, and then suddenly it doesn't, and then it does again.. and then doesn't.. without changing any code.. so the problem more seems to be an unreliable recaptcha solution, than any form of misuse of the code.. if it was as easy as using double id's there wouldn't be a problem..

https://stackoverflow.com/questions/55521795/google-recaptcha-v3-returns-success-as-false

I read this topic 2 months ago when I really needed help with my recaptcha. There's an easier way to apply recaptcha with php and javascript. Did you ever think of just rewriting the code? Or changing the format of the code? Perhaps think of a different method to get recaptcha to work because obviously, your first method isn't. I mean, you're a programmer. Problem solve. You're not going to be able to google everything, a lot of this stuff is testing things and figuring it out on your own. Just have to figure out what works best.

With that aside, there has to be a reasonable solution as to why the code is doing that. Been using recatpcha for months and it works fine. When was the last time you tested your code?

**Comment by jjxtra** [comment]
Seeing this as of today. Code has not changed in months, but suddently 0.1 score always. What is going on?

**Comment by quicirman** [comment]
A couple of days ago, I implanted reCAPTCHA V3 to my website and I am in tests.
All the tests I made from production, I do not use localhost.
The key validations are correct, the JSON response is correct (success = true), but I am having problems with the score. The tests are through a contact form, so I put action = contact. I fill in all the fields of the form by hand, check the box of Privacy Policy and submit. It does not matter if I use Chrome, Firefox or Opera, it always gives me score = 0.1, so I can not validate those cases where score <0.5 is a bot, because nobody would pass the filter.
I'm afraid I'm going to have to use V2, since I do not know how to solve this problem :(

**Comment by doekia** [comment]
All the tests I have done proves V3 is totally unreliable, not to mention it requires a complete revision of your business logic in many cases. You receive score varying from .9 to .1 over the same connection, same user agent, ... When you want to protect a form that requires end-user input it is even worst. You receive timeout - sorry end-user may takes more than 1mn to fill-up their name, address, phone, ... Implementing some cleaver logic to address such makes it spammer prone.
Some browser are definitly unsupported, the code is negatively impacting performances (loading various fonts, and other third party resources.
This code is useless, support non-existent, documentation fuzzy when not misleading...

**Comment by davit555** [comment]
Anyu updates here ?

**Comment by splpa** [comment]
I agree with @doekia what a mess. We like the idea of not needing user input, but consistency/reliability is a must obviously. And implementing this has been confusing for sure, receiving errors with practically no guidance on how to resolve, now scores are unreliable, feeling very frustrated.

**Comment by crmpicco** [comment]
The comments on here seem like it's RIP to reCAPTCHA v3. This issue was opened a year ago and it hasn't been resolved yet. 💀

**Comment by KMelnikoff** [comment]
We are experiencing the same issue in production. The automated traffic from the malicious sources (doing constant unlimited request from one IP) got 0.9 scores, meaning, reCaptcha v3 is USELESS.
There is no way to at least lower the score if the bot fraud traffic gets 0.9 all the time.
What a mess Google!

**Comment by andrebruton** [comment]
It looks like the solution is to either only use Ver 2, or use Ver 3 on initial page and then use Ver 2 on any other pages where Ver 3 does not work. I thought it would do this automatically, but clearly not.

**Comment by mostafaznv** [comment]
many people are talking about this problem. 
but there is no reaction from recaptcha team. 
what a support! 
thanks

**Comment by jjxtra** [comment]
Would be nice to have an official Google response...

**Comment by crmpicco** [comment]
![im out](https://i.imgur.com/jJmiG2h.gif)

**Comment by nuenuewei** [comment]
Why would Google respond? You're not the customer, you are an advertising/data product they sell to their customers.

**Comment by superern** [comment]
since this issue is still open. is there any other alternatives we can use for the meantime?

**Comment by duxiaofeng-github** [comment]
according to [this answer](https://stackoverflow.com/a/52170635/1010869), it seems embedded browsers will get lower score.

i tested it by myself. as the answer says, i visited my website by chrome and did the exact same action as my clients do, and i got 0.9.

but all of my normal clients visit my website by embedded browsers of wechat. how can i resolve this problem?

**Comment by duxiaofeng-github** [comment]
also, recaptcha not always return 0.1 in my case, few of our users got 0.3. so, i think it's still working.

**Comment by MakingStuffs** [comment]
Not sure if people are still having this issue but changing the action from homepage to submit fixed it for me. My check is not executed until the form is _submitted_however.

**Comment by MonDeveloper** [comment]
Bug Still present.
Since it probabily relies on the ML under the hood it is quite hard to find similarities but it's pretty sure it is bugged.
Same interaction made by IE11 and Chrome (both Incognito, both from the same PC) takes 0.1 and 0.3 using IE11 and 0.9 using Chrome.

**Comment by potsky** [comment]
Buggy today for me after several months ok :-(
Env : In production and on localhost.
IPs : From distinct networks
The values are always 0.1 or 0.3.

**Comment by davit555** [comment]
same here ((
very disappointed 
started to show low scores, mostly 0.1

**Comment by stalker780** [comment]
Same here. 3 domains on one VPS. Dozens of clients. All get 0.3-0.1 score since this morning.

Damn. Had to disable recaptcha :(

**Comment by Chrispybee** [comment]
I've been using recaptcha V3 now for just over a year and it's really flaky. At 4am on Thursday (15th April 2021) we started having an influx of users who would normally be getting scores of 0.7 - 0.9 were getting 0.1 - 0.3. The ramp of users getting a low scores has dropped slightly but still getting high numbers. These are genuine users who aren't using a bot to use the login page.

Does anyone know if there was there some sort of outage which was causing this issue? Just seems like it's getting worse and no help from Google to guide us to get our customers to 0.9 or the magical 1 score!

Looking at the documentation and the quick video regarding the enterprise offering, the admin console looks worse and it will be the same ML underneath, just get an email address for support that might work...

**Comment by beqramo** [comment]
same here :( everything was working perfectly but suddenly 0.1, 0.3

**Comment by felixck12** [comment]
I thought it was just us. We have been getting low scores mostly 0.1 as well since late March. Left a comment in issue #304

**Comment by dusnoki** [comment]
We are having the same problem. A couple of days ago we started getting lots of 0.1 scores. Even our own developers are getting bad scores. It's all a bit flakey, sometimes the score is good, sometimes not. It seems to be affecting the domains which are behind a CDN (Cloudfront) more than the ones that aren't. But it might be only a fluke. Had to disable recaptcha for now.

**Comment by capytan** [comment]
same here. It's still going on right now.

I've been using reCAPTCHA v3 since 2019 and have never had it used to work perfectly. We used to get scores of 0.7 or 0.9. However, there was a sudden increase in the number of requests evaluated as 0.1 or 0.3 since April 15. We found that these were normal requests made by humans.

**Comment by felixmeziere** [comment]
Same here! suddenly I started getting 0.1 and 0.3 on lots of calls randomly :/

**Comment by Chrispybee** [comment]
I do wish Google would come back to us and give us in the response why it failed. Just having a black box and getting an arbitrary number just seems flaky. I love the idea of V3 but where one user could get a 0.9 and the next second get a 0.3 just doesn't make any sense - almost like pointing a finger in the air and taking a guess.

Has anyone moved to V2 to see if that's any better?

**Comment by bankeris** [comment]
V2 and V3 are different things. V2 you have google UI, while V3 works silent :)

Using V3 on developer/server in debug mode sometimes(very rare) getting low score. But my main issue silently fixed.

**Comment by Chrispybee** [comment]
I get the difference between the two, this is why I went with the V3 solution. My question was, due to the amounts of low scores, has anyone just binned it off and gone for V2 instead. If so, what was their experience? Were their users fed up because of a captcha every time they visited the page? Is invisible captcha the way forward if the scoring mechanism is failing so badly..

**Comment by gingerlime** [comment]
V3 is a marketing tool to get you to also install V2 ;-)

**Comment by Xeyor** [comment]
Hi everyone same problem here we are using recaptcha v3 in production for months now, everything was working ok but since a few days we receive really low scores even for real users (Mostly 0.1 and 0.3)

My guess is that google messed up something on their end because I remember getting rejected by a recaptchaV3 few days ago on another website and you guys seems to have the problem too...

We need explanations !

**Comment by yoyoloftis** [comment]
Same issue for us. We are using recaptcha v3 on some of our website forms. On 4/21/21 (two days ago), we started experiencing super low scores of 0.1 and 0.3, even for real users. In testing myself, I am sometimes getting 0.7 score, and sometimes getting a 0.1 or a 0.3, depending on browser. Developer partners get same results. This is super frustrating, since we have set up some blockers and error messages for end users in these low score scenarios. This is bad for customer service. It would be good to know if Google has changed anything recently. Because we have not made any changes, and it was working great until this week.

**Comment by armellarcier** [comment]
Same here. We changed the action strings sent for each specific verification and our scores are good again.
So we appended a dynamic suffix to all action strings (the current date formatted YYYYMMDD).
We're hoping this will make our score consistently better.

**Comment by justinphan1992** [comment]
Any update for this issue. I'm facing this problem today

**Comment by MonDeveloper** [comment]
I strongly suggest all of you to migrate to hCaptcha, they are
highly compatible with reCaptcha but they care about the service.

I am not involved in any manner with hCaptcha, I didn't know about them
since 1 week ago and I'm neither a current hCaptcha customer, just did some
experiments.

It's very sad to notice how much Google ignores reCaptcha service.

Il giorno lun 31 mag 2021 alle ore 04:32 Phan Dat ***@***.***>
ha scritto:

**Comment by evdama** [comment]
As a firebase user I've enabled the recently announced app check feature... maybe it has something to do with it? 🤷‍♂️ 
https://firebase.google.com/docs/app-check/web#from-the-cdn

**Comment by gingerlime** [comment]
> I strongly suggest all of you to migrate to hCaptcha, they are highly compatible with reCaptcha but they care about the service.

Last I checked with hCaptcha, the invisible captcha option was only available to paid customers, and prices start at $999/mo ...

**Comment by benazo** [comment]
Ran into this today.  We've been using "www.recaptcha.net" on our live site rather than "www.google.com" due to "www.google.com" not working for users in China.  Recaptcha v3 was returning a score of 0.3 for several real people in the US.  We tried switching back to "www.google.com" and their scores are 0.9.

**Comment by dslocalsearch** [comment]
This is also happening to us. Even more weird, we tried to lower the score to 0.3 but then when we test with the googlebot header, it returns 0.3 and allows the request. This is very frustrating and unreliable solution in my opinion.

**Comment by Chrispybee** [comment]
We decided in the end to go down the route of Recaptcha v2. It’s a shame but v3 was completely unreliable for what we wanted.

**Comment by subbuswaroop** [comment]
I'm also facing the same issue for the past few days. It returns a low score of 0.1 when I test it in https://www.live.browserstack.com on iOS and iPad. But it returns a high score when I test the same in Google chrome on my windows machine.

The beavhiour is very inconsistent. Seriously hope Google fixes it sooner.

**Comment by antonio-acevedo** [comment]
We are also seeing strange behaviour on our enterprise recaptcha where legit requests from Iphones are being scored as 0.1 (or 0.3 if users happen to be logged in their google acocunt within the iphone's browser).

Working perfectly fine for all other OS and browsers...is anybody else seeing this behaviour?

**Comment by isanecat** [comment]
I also have the same problem when I am trying my code on localhost. The score returns 0.1 despite user handled interactions. It was working for months despite no changes on recaptcha code.

**Comment by ConaxLiu** [comment]
We also received reports that some frustrated users constantly receiving low score, beween 0, 0.1 and 0.3.
Some had tried different browsers with none success.
With the information received so far, I expect its something to do with the VPN their organisations are using, or the policies configured on their work computer. (One user had indicated that she was blocked by recaptcha (v3) on her work computer, but was able to progress if using her personal computer.

**Comment by laur89** [comment]
Recently started having issues with logging onto multiple services that employ recaptcha. Both Chromium & Firefox fail because captcha scoring is stable from 0.1-0.3. Switch to Google Chrome and suddenly you're not deemed a bot with 0.9.
Verified it also on [this page](https://democaptcha.com/demo-form-eng/recaptcha-3-test-score.html)

One can hope EU will take a note of this shit at one point.

**Comment by brookeherbert** [comment]
We are also out of the blue getting 0.1 scores from legitimate safari users, using the enterprise version. We could try updating the key or action name as others have mentioned, but this seems to be an arbitrary solution. Considering falling back to v2

**Comment by Chrispybee** [comment]
Personally, it doesn’t matter what you change/ workaround you’ll always end up with users not being able to login due to low scores.Each and every time is to use v2. Not had any issues with it since implementing it.Sent from my iPhoneOn 11 Apr 2023, at 00:55, brookeherbert ***@***.***> wrote:﻿
We are also out of the blue getting 0.1 scores from legitimate safari users, using the enterprise version. We could try updating the key or action name as others have mentioned, but this seems to be an arbitrary solution. Considering falling back to v2

**Comment by ConaxLiu** [comment]
Ya, I ended up implementing both V2 and V3, so those that can use V3 continue to enjoy normal login, and those that are blocked by V3 are then presented with V2 images. 😅

**Comment by hugopeek** [comment]
> Closing old issues that are not related to the PHP client code.

@rowan-m What is it related to then?

**Comment by Namstel** [comment]
So I'm too noticing this when testing.

I'm on Firefox and I get low scores (0.3), I go on Chrome and the scores are fine (0.9). 

I do everything by hand, moving the cursors to the form fields, typing out the value instead of using autocomplete, but still Firefox returns 0.3 and Chrome returns 0.9.

Perhaps the privacy enhancing features of Firefox are preventing certain telemetry being measured that's being used by reCAPTCHA?

**Comment by craigs100** [comment]
I've just encountered this out of the blue as well on local host. For testing you could be sending hundreds of tests in with the same form data. It's ridiculous that local host is treated the same as a live domain!  Will just take it out for now as it's defeating the whole object. Well done Google.

**Comment by coltmans** [comment]
We saw this now also happening to safari users getting a score below 0.5.
When they delete the page cache they pass again, but another check afterwards fails again.
